src/main/java/com/currencybaskets/tasks/UpdateLatestRateScheduledTask.java

```diff
@@ -0,0 +1,51 @@
+package com.currencybaskets.tasks;
+
+import com.currencybaskets.dao.repository.RateRepository;
+import com.currencybaskets.dto.fixer.CurrencyRate;
+import com.currencybaskets.dto.fixer.LatestRate;
+import com.currencybaskets.exceptions.ServiceUnavailableException;
+import com.currencybaskets.services.AccountService;
+import com.currencybaskets.services.CurrencyFixerClient;
+import com.currencybaskets.view.RateUpdate;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Component
+public class UpdateLatestRateScheduledTask {
+
+	private static final Logger LOG = LoggerFactory.getLogger(UpdateLatestRateScheduledTask.class);
```

Comment:
Please use Lombok as in this class https://github.com/egorvlgavr/currency-baskets/blob/master/src/main/java/com/currencybaskets/services/AccountService.java#L20